renovate.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
```

Comment:
we said in: https://github.com/Pix4D/renovate-bot-pci/pull/4 that this is not needed...